editor/src/components/inspector/sections/component-section/component-section.spec.browser2.tsx

```diff
@@ -835,11 +835,22 @@ describe('Controls from registering components', () => {
         Link: {
           preferredChildComponents: [
             {
-              additionalImports: undefined,
               name: 'span',
               variants: [
                 {
-                  code: '<span>Link</span>',
+                  importsToAdd: {
+                    '/utopia/storyboard': {
+                      importedAs: null,
+                      importedFromWithin: [
+                        {
+                          alias: 'span',
+                          name: 'span',
+                        },
+                      ],
+                      importedWithName: null,
+                    },
+                  },
```

Comment:
This is unexpected. Why are we importing `span` from the storyboard

Reply:
Thanks for catching this, I updated the code so that no imports are added for built-in elements https://github.com/concrete-utopia/utopia/pull/5050/commits/6f71152ea5296bbb221507c86e78878e3b73ed56